courses/phi305/quizz/009/fr.yml

```
@@ -1,4 +1,4 @@
-question: La famille de Spinoza était-elle ?
+question: La famille de Spinoza était-elle de quelle confession?
 answer: De confession juive.
```

```suggestion
answer: Juive.
```
Consistency with the other answers